hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstantTimeGenerator.java

```diff
@@ -88,6 +88,18 @@ public static String createNewInstantTime(boolean shouldLock, TimeGenerator time
     });
   }
 
+  public static String formatMillis(long milliseconds) {
```

Comment:
This seems to be a copy of org.apache.hudi.common.table.timeline.HoodieInstantTimeGenerator#createNewInstantTime. can we try to reuse the existing code? If the HoodieInstantTimeGenerator implementation is changed, then we will need to update this too